test/run-spec-wasm2c.py

```diff
@@ -616,24 +623,28 @@ def main(args):
             wasm2c.RunWithArgs(wasm_filename, '-o', c_filename_input, *args)
             if options.compile:
                 for j, c_filename in enumerate(c_filenames):
-                    o_filenames.append(Compile(cc, c_filename, out_dir, *cflags))
+                    o_filenames.append(Compile(cc, c_filename, out_dir, use_c11, *cflags))
 
         cwriter.Write()
         main_filename = utils.ChangeExt(json_file_path, '-main.c')
         with open(main_filename, 'w') as out_main_file:
             out_main_file.write(output.getvalue())
 
         if options.compile:
-            # Compile wasm-rt-impl.
-            wasm_rt_impl_c = os.path.join(options.wasmrt_dir, 'wasm-rt-impl.c')
-            o_filenames.append(Compile(cc, wasm_rt_impl_c, out_dir, *cflags))
+            # Compile runtime code
+            source_files = [
+                main_filename,
+                os.path.join(options.wasmrt_dir, 'wasm-rt-impl.c'),
+                os.path.join(options.wasmrt_dir, 'wasm-rt-exceptions-impl.c'),
+                os.path.join(options.wasmrt_dir, 'prebuilt', 'wasm-rt-mem-impl.c'),
```

Comment:
Assuming @keithw doesn't object I'd rather drop the `prebuilt` here and just use the pre-processor to make this work.

Reply:
No objection from me; just the mild preference make this and whatever we do for tables match each other (https://github.com/WebAssembly/wabt/pull/2308#discussion_r1465921700).

Reply:
I'll follow with the change to tables in a separate PR to keep this one clean